src/pages/Selector/container.jsx

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
+import { transformEmployees as transform } from './transform-employees';
 
-// temporarily disabling this rule until the function is fleshed out
-/* eslint-disable react/prefer-stateless-function */
 const SelectorContainer = Wrapped =>
-  class Selector extends React.Component {
+  class extends React.Component {
+    state = { employees: [] };
+
+    componentDidMount() {
+      this.fetchEmployees();
+    }
+
+    fetchEmployees = () => {
+      fetch('http://localhost:8081/api/users').then(resp => {
+        resp.json().then(json => this.setState({ employees: json }));
```

Comment:
I assume we're going to want to parse this as JSON everytime we use the fetch endpoint? Maybe write a wrapper function around this that will do this rather than calling resp.json in each container? Hopefully that makes sense.

Also we might want to change the variables here. resp.json() returns a promise containing the employees json object. Maybe employees could be the variable and then we could just do `this.setState({ employees });`